express-post-json/index.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+
+const app = express();
+
+const grades = {
+
+};
+
+app.get('/api/grades', (req, res) => {
+  const arr = [];
+  for (var key in grades) {
+    arr.push(grades[key]);
+  }
+  res.json(arr);
+});
+
+app.listen(3000, () => {
+  // eslint-disable-next-line no-console
+  console.log('Express server listening on port 3000');
+});
+
+app.use(express.json());
+
+app.post('/api/grades', (req, res) => {
+  grades[req.params.id] = req.params;
+  res.sendStatus(201);
```

Comment:
Here you should be using the data from `req.body` to create a new grade object that contains `name`, `course`, `score`, and `id`. Then new grades object should be added to the `grades` object using the `id`. Then the new grade should be sent as the response to the client.